src/pages/dataTableUtilities/getPageInitialiser.js

```diff
@@ -0,0 +1,213 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+import { UIDatabase } from '../../database';
+import Settings from '../../settings/MobileAppSettings';
+
+import { newSortDataBy, getAllPrograms } from '../../utilities';
+import { recordKeyExtractor } from './utilities';
+
+const customerInvoiceInitialiser = pageObject => ({
+  pageObject,
+  backingData: pageObject.items,
+  data: pageObject.items.sorted('item.name').slice(),
+  keyExtractor: recordKeyExtractor,
+  dataState: new Map(),
+  searchTerm: '',
+  filterDataKeys: ['item.name'],
+  sortBy: 'itemName',
+  isAscending: true,
+  modalKey: '',
+  modalValue: null,
+  hasSelection: false,
+});
+
+const customerInvoicesInitialiser = () => {
```

Comment:
do you think these need doc strings or ott

Reply:
🤣 . 98bc3c103448b3fcef6587e2ff3c600f89eeb246.